assets/prefabs/food/raw/Pecan.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:items#pecan",
```

Comment:
Remove "items#" from this line. This is only used when there is an icon atlas being used